src/ast.rs

```diff
@@ -423,6 +425,10 @@ fn analyze_dependencies(
     .collect()
 }
 
+fn filter_dep_comment(c: &deno_ast::dep::DependencyComment) -> bool {
+  c.text.contains("@deno-types")
+}
```

Comment:
Please make sure I have this right. These leading comments seem to only be used for `@deno-types` resolution.

(Side note: there is already a test that fails when this is some other random string)